Weater-app/src/components/historyModal/styledComponents/HistoryModal.js

```diff
@@ -0,0 +1,102 @@
+import styled from 'styled-components';
+
+import {
+  Black,
+  DarkGray,
+  DodgerBlue,
+  LightBlack,
+  White,
+} from '../../../assets/colors/colors';
+
+export const lightTheme = {
```

Comment:
i think i got it, you export themes for each component...
it's bad idea in my opinion - better to keep it in single file and reuse everywhere, because it's a lot of code duplication